pterodactyl/remote/server_backups.md

```diff
@@ -0,0 +1,89 @@
+### `GET /api/remote/backups/:backup`
+
+Returns the required presigned urls to upload a backup to S3 cloud storage.
+
+### Parameters
+
+| Name      | Visibility | Description                        |
+| --------- | ---------- | ---------------------------------- |
+| size      | optional   | The backup file size.              |
+
+### Responses
+
+| Code | Description                                                    |
+| ---- | -------------------------------------------------------------- |
+| 200  | The response was successful.                                   |
+| 400  | The configured backup adapter is not an S3 compatible adapter. |
+| 404  | The backup was not found.                                      |
+| 409  | The backup is already in completed state.                      |
+
+### Response examples
+
+WIP.
+
+Sources
+
+- [app/Http/Controllers/Api/Remote/Backups/BackupRemoteUpl2oadController.php#L33](https://github.com/pterodactyl/panel/blob/v1.11.3/app/Http/Controllers/Api/Remote/Backups/BackupRemoteUploadController.php#L33)
+
+---
+
+### `POST /api/remote/backups/:backup`
+
+Handles updating the state of a backup.
+
+### Body
+
+| Name  | Visibility | Type   | Description                                                                                            |
+| ----- | ---------- | ------ | ------------------------------------------------------------------------------------------------------ |
+| data  | required   | object | An object containing the checksum, checksum type, backup size, success state, and parts of the backup. |
```

Comment:
The actual fields are in [ReportBackupCompleteRequest.php](https://github.com/pterodactyl/panel/blob/b23f3114e4cb71406e6de17d0dca0d88205f56c1/app/Http/Requests/Api/Remote/ReportBackupCompleteRequest.php).

Reply:
I don't know what `etag` and `part_number` does, but I committed "???" as a placeholder.

Reply:
I'll resolve this anyways for now.